src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -306,6 +364,8 @@ public void When_Style_ControlTemplate()
 			var r = Windows.UI.Xaml.Markup.XamlReader.Load(s) as UserControl;
 			Assert.IsNotNull(r);
 
+			r.ForceLoaded();
```

Comment:
I'm always a bit suspicious when this needs to be added to an existing test... is this consistent with Windows?

Reply:
This is needed because as with the rest of the `ResourceResolver.ApplyResource` invocations, we need to go through the Loading event to resolve lazy static/theme resources.

Reply:
that doesn't actually answer my question but fine :p